packages/optimizer/src/lib.rs

```diff
@@ -1,16 +1,22 @@
-pub mod builtins;
-
 use log::Level;
 use serde::{Deserialize, Serialize};
 use ts_rs::TS;
 use wasm_bindgen::{prelude::wasm_bindgen, JsValue};
 
 type Bool = i32; // 0 or 1 (`wasm-bindgen` doesn't support `bool` well)
 
-type Vector = nalgebra::DVector<f64>;
+#[wasm_bindgen(module = "/call.js")]
```

Comment:
Can you explain what `call.js` does and what the `extern "C"` means here? I can see it generates a `call.js` file but not sure what it's for.

Reply:
I added a comment attempting to explain this; hopefully it clarifies?

Reply:
Gotcha. Weird that `extern "C"` is the standard lol but great to know about the `call4` motivation.